packages/styling/src/styles/DefaultFontStyles.ts

```diff
@@ -130,7 +130,6 @@ function _createFont(size: string, weight: number): IRawStyle {
   return {
     fontFamily: _getFontFamily(),
     MozOsxFontSmoothing: 'grayscale',
-    MsHighContrastAdjust: 'none',
```

Comment:
It was removed for Fabric Core 7.1.0 in [this PR](https://github.com/OfficeDev/office-ui-fabric-core/pull/988) so, yes, it should be removed from the Styling package as well.